docs/project/find-an-issue.md

```diff
@@ -42,68 +42,94 @@ weight=2
 
 # Find and claim an issue
 
-On this page, you choose the issue you want to work on. As a contributor, you can work
-on whatever you want. If you are new to contributing, you should start by
-working with our known issues.
+As a contributor, you can work on any open issue you want. You can view issues in the Issues tab in every repository.
+If you are new to contributing, use the filter option to find suitable issues. You can filter issues by Author, Labels, Milestones, and Assignee or sort by time such as newest and oldest. 
 
 ## Understand the issue types
+To help you identify the different types of issues, our maintainers assign labels to issues. Labels are color-coded and help you categorize and filter issues. There are four labels categories, kind, area, experience, and priority. You can filter using one or more labels. The kind and experience labels are useful for new contributors.
```

Comment:
should the comma be a semi-colon after "categories"?

also, could you wrap this paragraph to 80 chars?